MathlibTest/Simproc/Divisors.lean

```diff
@@ -0,0 +1,41 @@
+import Mathlib.Tactic.Simproc.Divisors
+import Mathlib.NumberTheory.Primorial
+
+open Nat
+
+example : Nat.divisors 1710 = {1, 2, 3, 5, 6, 9, 10, 15, 18, 19, 30, 38, 45, 57,
+      90, 95, 114, 171, 190, 285, 342, 570, 855, 1710} := by
+  simp only [Nat.divisorsEq]
+
+example : Nat.divisors 57 = {1, 3, 19, 57} := by
+  simp only [Nat.divisorsEq]
+
+example : (Nat.divisors <| 6 !).card = 30 := by
+  simp [Nat.divisorsEq, Nat.factorial_succ]
+
+example : (Nat.divisors <| primorial 12).card = 32 := by
+  --TODO(Paul-Lez): seems like we need a simproc for computing `primorial`!
+  have : primorial 12 = 2310 := rfl
```

Comment:
This looks more `norm_num` shaped to me.

Reply:
Edited the comment:)